spec/requests/dashboard_request_spec.rb

```diff
@@ -2,9 +2,38 @@
 
 RSpec.describe "DashboardController" do
   describe "GET /" do
+    let(:user) { create_and_log_in_user }
     before { get "/" }
 
     it_behaves_like "a page with footer content"
+
```

Comment:
can we add the header content examples too?
```suggestion
    it_behaves_like "a page with footer content"
    it_behaves_like "a page with header content"

```

Reply:
can you also add the `header content` and `utility menu links` examples to the describe block for `GET /dashboard`, just so it's tested on multiple routes?